sympy/physics/vector/tests/test_functions.py

```diff
@@ -488,3 +490,22 @@ def test_partial_velocity():
 
     raises(TypeError, lambda: partial_velocity(Dmc.vel(N), u_list, N))
     raises(TypeError, lambda: partial_velocity(vel_list, u1, N))
+
+
+def test_different_time_symbol_old():
+    x = Symbol('x')
+    dynamicsymbols._t = x
+    q = dynamicsymbols('q')
+    assert q.diff(x) == Derivative(Function('q')(x), x)
```

Comment:
This needs a try/finally as well to restore the time symbol for subsequent tests